.github/workflows/ci.yml

```diff
@@ -29,12 +29,19 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Build and test artifacts
         run: ./gradlew build
-      - name: Deploy artifacts
-        run: ./gradlew build artifactoryPublish -x test
+        env:
```

Comment:
Consider removing the artifactory variables earlier in the file
e.g. `ARTIFACTORY_PUBLISH_USER`